test/jdk/java/lang/invoke/lambda/LogGeneratedClassesTest.java

```diff
@@ -207,15 +209,16 @@ private static boolean isWriteableDirectory(Path p) {
 
     @Test
     public void testDumpDirNotWritable() throws Exception {
-        if (!Files.getFileStore(Paths.get("."))
-                  .supportsFileAttributeView(PosixFileAttributeView.class)) {
+        FileStore fs;
+        try {
+            fs = Files.getFileStore(Paths.get("."));
+        } catch (IOException e) {
+            e.printStackTrace();
+            throw new SkipException("WARNING: IOException occur. Skipping testDumpDirNotWritable test.");
```

Comment:
Nit: "occurred" instead "occur". Additionally, I would suggest even the exception's toString() in that message just to provide additional context at the location wherever this will get reported outside of a .jtr (if at all). So something like:
```java
throw new SkipException("WARNING: IOException occurred: " + e + ", Skipping testDumpDirNotWritable test.");
```

Reply:
Thanks for your advice and review again.